zndraw/server/events.py

```diff
@@ -663,8 +667,9 @@ def modifier_available(available: bool):
 @io.on("modifier:queue:update")
 def modifier_queue_update(data: dict):
     """Update the modifier queue."""
-    queue_position = get_queue_position(data["queue_name"], data["job_id"])
-    emit("modifier:queue:update", queue_position, to=f"webclients_{session['token']}")
+    queue_positions = get_queue_position(data["queue_name"])
```

Comment:
do we allow read-only database access in `events`? This might potentially be blocking.